packages/at_secondary_server/pubspec.yaml

```diff
@@ -35,6 +35,14 @@ dependencies:
   yaml: 3.1.2
   logging: 1.2.0
 
+dependency_overrides:
+  at_persistence_secondary_server:
+    git:
+      url: https://github.com/atsign-foundation/at_server.git
+      path: packages/at_persistence_secondary_server
+      ref: apkam_sync_persistence_changes
```

Comment:
Please remove dependency override

I note that the `apkam_sync_persistence_changes` branch looks a bit stale - if it is not an active branch then it should be deleted.

Reply:
Removed dependency overrides.